ansible_image_validation/validation-playbooks/per-vm-validation.yaml

```diff
@@ -60,19 +60,13 @@
   ignore_errors: yes
   when: ansible_os_family == "RedHat"
 
-- name: Check the kernel version
-  shell: uname -r
-  register: output_kernel_version
-
-- debug:
-    var: output_kernel_version.stdout_lines 
+- name: Print all available facts
+  ansible.builtin.debug:
+    var: ansible_facts["kernel"]
 
 - name: Set the kernel version
   set_fact: 
-    kernel_version: "{{ output_kernel_version.stdout_lines | join}}"
-
-- debug:
-    var: kernel_version
+    kernel_version: "{{ ansible_facts['kernel'] | join}}"
```

Comment:
join is not needed i think. is the output a list?

Reply:
Check this https://msazure.visualstudio.com/One/_git/Compute-Runtime-Tux-RHUI?path=/ansible/rhui3/rhui3-update-kernel.yml&version=GBmaster&line=53&lineEnd=53&lineStartColumn=18&lineEndColumn=31&lineStyle=plain&_a=contents

Reply:
Test your code too